secboot/secboot_tpm_test.go

```diff
@@ -1153,24 +1155,96 @@ func (s *secbootSuite) TestUnlockEncryptedVolumeUsingKeyErr(c *C) {
 	})
 }
 
-func (s *secbootSuite) TestUnlockVolumeUsingSealedKeyIfEncryptedFdeRevealKey(c *C) {
-	n := 0
+func (s *secbootSuite) TestUnlockVolumeUsingSealedKeyIfEncryptedFdeRevealKeyErr(c *C) {
+	// this test uses a real systemd-run --user so check here if that
+	// actually works
```

Comment:
Nitpick, maybe add a comment explaining that we need real systemd-run in order to test the status of failing hook.

Reply:
I will do this (and the others) in a followup, I have a PR stacked on top of this one that is ready :)